src/hotplug_libusb.c

```diff
@@ -549,6 +549,7 @@ LONG HPSearchHotPluggables(void)
 {
 	int i;
 
+	AraKiriHotPlug = FALSE;
```

Comment:
Hi, what do you think about this change? I understand it's not strictly needed for the upstream PCSC project (as it doesn't have unit tests of such kind, which is what I'm trying to add in our fork at https://github.com/GoogleChromeLabs/chromeos_smart_card_connector), but this change seems innocent enough anyway.